src/Parser/Mage/Fire/Modules/Features/Abilities.js

```diff
@@ -33,6 +46,7 @@ class Abilities extends CoreAbilities {
       {
         spell: SPELLS.FIRE_BLAST,
         category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+        isOnGCD: false,
```

Comment:
it defaults to false so no need to add it

Reply:
i know it defaults to false, but for the sake of completeness i opted for including it anyway so at a glance you could look at the spellbook and get all the info about that spell including that its off the gcd.

Granted you could just assume that its off the gcd if the item was missing, but this just seemed to be more clear and doesnt really hurt anything to have it explicitly marked false.